mac-cleanup

```diff
@@ -118,11 +118,14 @@ msg 'Clearing System Cache Files...'
 sudo rm -rfv /Library/Caches/* &>/dev/null
 sudo rm -rfv /System/Library/Caches/* &>/dev/null
 sudo rm -rfv ~/Library/Caches/* &>/dev/null
+sudo rm -rfv /private/var/folders/bh/5qn6f27x049drtr8rqq75jt80000gp/C/*/* &>/dev/null
```

Comment:
Are you sure that THIS folder will be on every machine?!

Reply:
You're right, I should just remove the general temp file directory instead... That folder has been on my system for as long as I can remember, so I must have been biased.